backend/app/application/templates/privacypolicy.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Certificate Portal Privacy Policy</title>
+	</head>
+	<body>
+		<h1>Certificate Portal Privacy Policy</h1>
+
+		<p>The Certificate Portal is part of Warwick Data Science Society (WDSS). This privacy policy will explain how our organisation uses the personal data we collect from you when you use our website. Warwick Data Science Society will be referenced as WDSS in the policy from this point onwards.</p>
+		<p>Topics:
+		<dl>
+			<li>What data do we collect?</li>
+			<li>How do we collect your data?</li>
+			<li>How will we use your data?</li>
+			<li>How do we store your data?</li>
+			<li>What are your data protection rights?</li>
+			<li>What are cookies?</li>
+			<li>How do we use cookies?</li>
+			<li>What types of cookies do we use?</li>
+			<li>How to manage your cookies</li>
+			<li>How to contact us</li>
+			<li>How to contact the appropriate authorities</li> </dl></p>
+		<h2>What data do we collect?</h2>
+		<p>Our society collects the following data:
+		<dl>
+			<li>Personal identification information:</li>
+			<dl>
+				<li>Name</li>
+				<li>Email address</li>
+			</dl>
+		</dl></p>
+
+		<h2>How do we collect your data?</h2>
+		<p>When you enrol on a WDSS course, we collect and process personal data. No extra personal data is processed when using this site.
+		</p>
+          
+		<h2>How will we use your data?</h2>
+		<p>WDSS collects your data so we can:
+		<dl>
+			<li>Generate a course certificate</li>
+			<li>Allow mentors to access the portal</li>
+			<li>Email you if there are any issues</li>
+		</dl></p>
+
+		<h2>How do we store your data?</h2>
+		<p>WDSS securely stores your data on our cloud server hosted by DigitalOcean. This is only accessible via ssh keys that require the root (admin) account to approve. DigitalOcean also enforces 2FA on all accounts. Data is then stored on a database which requires a separate admin password to access.</p>
+
+		<p>WDSS will keep your personal information so the certificate can be accessed at any time.</p>
```

Comment:
https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/principles/storage-limitation/

Think you need to state personal data will be kept no longer than necessary, rather than just saying it will be kept. A retention period of some sort needs to be specified